src/containers/AudioUploader/components/AudioManuscript.tsx

```diff
@@ -61,22 +75,104 @@ const manuscriptPlugins: SlatePlugin[] = [
+  const { userPermissions } = useSession();
+  const toast = useToast();
+  const [isPolling, setIsPolling] = useState<boolean>(false);
+  const [_field, _meta, helpers] = useField("manuscript");
+
+  const language = LANGUAGE_MAP?.[audioLanguage] ?? LANGUAGE_MAP.nb;
+  const postAudioTranscriptionMutation = usePostAudioTranscriptionMutation();
+  const fetchAudioTranscriptQuery = useAudioTranscription(
+    {
+      audioId: audio?.id ?? -1,
+      language: language,
+    },
+    {
+      // Settings to make this a lazyQuery
+      enabled: false,
+      retry: false,
+    },
+  );
+
+  const { data: polledData } = useAudioTranscription(
+    {
+      audioId: audio?.id ?? -1,
+      language: language,
+    },
+    {
+      refetchInterval: 1000,
+      enabled: isPolling && !!audio,
+    },
+  );
+
+  const startTranscription = async () => {
+    if (!audio || !audioLanguage) {
+      return;
+    }
+
+    const transcript = await fetchAudioTranscriptQuery.refetch({ cancelRefetch: false });
+
+    if (transcript?.data?.status === "COMPLETED") {
+      // TODO: use object directly when type is properly typed from backend
+      const transcriptText = parseTranscript(transcript?.data?.transcription ?? "");
+      const editorContent = inlineContentToEditorValue(transcriptText, true);
+      helpers.setValue(editorContent, true);
+      setStatus({ status: "manuscript" });
+    } else if (transcript?.data?.status === "FAILED") {
+      toast.error({ title: t("textGeneration.failed.transcription") });
+    } else {
+      const name = audio.audioFile.url?.split("audio/files/")[1];
+      await postAudioTranscriptionMutation
+        .mutateAsync({ name, id: audio.id, language })
+        .then(() => setIsPolling(true))
+        .catch((err) => {
+          if (err.status === 400 && err.json.code === "JOB_ALREADY_FOUND") {
+            setIsPolling(true);
+          }
+        });
+    }
+  };
+
+  useEffect(() => {
+    if (polledData?.status === "COMPLETED" && isPolling) {
+      setIsPolling(false);
+      const transcriptText = parseTranscript(polledData?.transcription ?? "");
+      const editorContent = inlineContentToEditorValue(transcriptText, true);
+      helpers.setValue(editorContent, true);
+      setStatus({ status: "manuscript" });
+    } else if (polledData?.status === "FAILED" && isPolling) {
+      setIsPolling(false);
+      toast.error({ title: t("textGeneration.failed.transcription") });
+    }
+  }, [helpers, isPolling, polledData?.status, polledData?.transcription, setStatus, t, toast]);
 
   return (
     <FormField name="manuscript">
-      {({ field, meta, helpers }) => (
+      {({ meta, helpers, field }) => (
         <FieldRoot invalid={!!meta.error}>
           <ContentEditableFieldLabel textStyle="title.medium">
             {t("podcastForm.fields.manuscript")}
           </ContentEditableFieldLabel>
           <RichTextEditor
+            id="manuscript"
```

Comment:
Vil ikke dette ødelegge for komposisjon med `Field`? De andre `Field`-komponentene regner vel med at ID'en har samme format som resten av `Field`-komponentene. Hvis vi trenger dette må vi nok sende det inn med et annet navn, f.eks `editorId`